packages/apps-v5/README.md

```diff
@@ -13,7 +13,7 @@ $ npm install -g @heroku-cli/plugin-apps-v5
 $ heroku COMMAND
 running command...
 $ heroku (--version)
-@heroku-cli/plugin-apps-v5/8.6.0 darwin-x64 node-v16.19.0
+@heroku-cli/plugin-apps-v5/8.8.0 darwin-arm64 node-v16.19.0
```

Comment:
I guess this also changes because of differences in the arch of the different dev boxes from each developer, isn't it?

Reply:
Yes, it is. Unfortunately.